app_dart/lib/src/service/config.dart

```diff
@@ -83,6 +83,13 @@ class Config {
     return defaultBranches[slug] ?? kDefaultBranchName;
   }
 
+  // GoB retry options
+  static const RetryOptions gobRetryOptions = RetryOptions(
+    maxAttempts: 4,
+    maxDelay: Duration(seconds: 1),
+    delayFactor: Duration(milliseconds: 300),
```

Comment:
Should we also add the randomizationFactor in here to keep things spicy?

Reply:
I believe there is a default randomization factor that is set to 25% as the default. Do you want to go higher than that? My take is that the higher the factor the longer the wait in between the calls.

Reply:
Thats fine, we can stick with default